imap_processing/tests/spice/test_geometry.py

```diff
@@ -306,3 +308,30 @@ def test_instrument_pointing(furnish_kernels):
             et, SpiceFrame.IMAP_HI_90, SpiceFrame.ECLIPJ2000, cartesian=True
         )
         assert ins_pointing.shape == (3, 3)
+
+
+@pytest.mark.external_kernel()
+@pytest.mark.use_test_metakernel("imap_ena_sim_metakernel.template")
+def test_basis_vectors():
+    """Test coverage for basis_vectors()."""
+    # This call to SPICE needs to be wrapped with `ensure_spice` so that kernels
+    # get furnished automatically
+    et = ensure_spice(spice.utc2et)("2025-09-30T12:00:00.000")
+    # test input of float
+    sc_axes = basis_vectors(et, SpiceFrame.IMAP_SPACECRAFT, SpiceFrame.IMAP_SPACECRAFT)
```

Comment:
Is there reason why both from and to frames are same? Just curious.

This place may be good to test one of the instrument frame. Eg. from S/C to an instrument frame?

Reply:
Yeah, I just wanted to test that keeping the same frame returned identity. Since this is a simple wrapper on the `get_rotation_matrix` function, adding more complicated tests here is discouraged IMO.

Reply:
I see. You know more about this. Sounds good to me